guide/students/student_management_service.bal

```diff
@@ -0,0 +1,322 @@
+    // Insert data to SQL database by invoking update action.
+    var ret = testDB->update(sqlString, name, age, mobNo, address);
+
+    // Use match operator to check the validity of the result from database.
+    match ret {
+        int updateRowCount => {
+         var result = getId(untaint mobNo);
+            // Getting info of the student added
+            match result {
+                table dt => {
+                    while (dt.hasNext()) {
+                        var ret2 = <Student>dt.getNext();
+                        match ret2 {
+                            // Getting the  id of the latest student added.
+                            Student stu => uid = stu.id;
+                            error e => io:println("Error in get employee from table: " + e.message);
+                        }
+                    }
+                }
+                error er => {
+                    log:printError(er.message,err = er);
+                }
+            }
+            updateStatus = { "Status": "Data Inserted Successfully", "id": uid };
+        }
+        error err => {
+            updateStatus = { "Status": "Data Not Inserted", "Error": err.message };
+        }
+    }
+    return updateStatus;
+}
+
+// Function to delete student data from database.
+  # `deleteData()` is a function to delete a student's data from student records database.
+  # + stuId - This is the id of the student to be deleted.
+  # + return - This function returns a json object. If data is deleted it returns json containing a status.
+  #              If data is not deleted , it returns the json containing a status and error message.
+
+public function deleteData(int stuId) returns (json) {
+    json status = {};
+    string sqlString = "DELETE FROM student WHERE id = ?";
+
+    // Delete existing data by invoking update action.
+    var ret = testDB->update(sqlString, stuId);
+    io:println(ret);
+    match ret {
+        int updateRowCount => {
+            if (updateRowCount != 1){
+                status = { "Status": "Data Not Found" };
+            }
+            else {
+                status = { "Status": "Data Deleted Successfully" };
+            }
+        }
+        error err => {
+            status = { "Status": "Data Not Deleted", "Error": err.message };
+            log:printError(err.message,err = err);
+        }
+    }
+    return status;
+}
+
+# `getId()` is a function to get the Id of the student added in latest.
+# + mobNo - This is the mobile number of the student added which is passed as parameter to build up the query.
+# + return -  This function returns either a table which has only one row of the student details or an error.
+
+// Function to get the generated Id of the student recently added.
+public function getId(int mobNo) returns (table|error) {
+//Select data from database by invoking select action.
+var ret2 = testDB->select("Select * FROM student WHERE mobNo = " + mobNo, Student, loadToMemory = true);
```

Comment:
formatting issues? 

Also we don't need the brackets around the return type if it is a union type. We can say
```ballerina
public function getId(int mobNo) returns table|error {
}
```

Reply:
Fix in other place too. Brackets are only needed with tuple types.

Reply:
Noted. I will make the necessary changes.